src/styles/components/footer.styl

```diff
@@ -4,6 +4,9 @@ footer
   background-color: black
   background: #272727
   color: hsla(0,0%,100%,.5)
+  position absolute
```

Comment:
This should be ok for now to put the footer at the bottom (not a blocker for me), but `position: absolute` will keep the footer in the window always.